### DISCUSSION TITLE
CI tutorial

### DISCUSSION THREAD

**Discussion by cordelia-g** [skills/.github#16] [General]
Hi there,

I'm at the point in creating a workflow in the CI tutorial and I got to this step:

Update the code in the ci branch to get the test to pass. You need to look something like this:

_underscore_

Commit changes.

Wait about 20 seconds then refresh this page for the next step.

Where do you put the _underscore_ in the ci.yml file?

**Comment by sinsukehlab** [comment]
Check the error in the ci.yml build and you will find a helpful warning message.

**Comment by housten** [comment]
1. I spent some time being confused in this same section and came here looking for hints. I got confused because the text in section 2 of the tutorial readme talks about looking for failing test names and the accompanying picture also illustrates a test log - but there aren't any tests being run in our ci yml file, There is just a lint warning. Then at the end of the section it talks about underscore so I looked more carefully at what my lint log said instead of trying to match my results to the rest of the text in the section. 
It is quite confusing - has the tutorial been changed perhaps and the readme not entirely updated?

2. Then, in section 3 we add an upload step but I get a warning about no files being found.  I thought this step would upload the lint report as a file but then the upload artifact doesn't happen on a lint failure and fails on pass because there is no public/ . Is this upload step related to test results that don't exist after changes to this tutorial?  It is useful to learn about the upload - but it would also be nice to see how it looks when a file does get uploaded...

3. In section 4 it says:
`Check all build and test jobs that you'd like to see in the newly visible gray box.`
But I could not find a list of build or test jobs on that page after selecting the two branch policy options. Is this a remant of a previous version too?

**Comment by bray2sick** [comment]
@cordelia-g in the branch `ci` that you just created, inside `resume.md` replace the asterisks `*` with underscores `_` and commit the changes. don't commit on the main branch.

@housten ignore that and create the branch protection rule to proceed to step 5
